pkgs/development/python-modules/bracex/default.nix

```
@@ -8,13 +8,13 @@
 
 buildPythonPackage rec {
   pname = "bracex";
-  version = "2.3.post1";
+  version = "2.4";
   format = "pyproject";
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/facelessuser/bracex/blob/9c9e5465f1a6d6e40763c9a2501b47a039909c30/pyproject.toml#L12